application/controllers/cron/Process.php

```diff
@@ -94,8 +94,9 @@ private function process_treebank($treebank)
 			foreach ($dirs as $dir)
 			{
 				// Create a Component for each directory in the .zip-file.
```

Comment:
Could be made into a function that creates a component

Reply:
Changed it!